src/components/Profile/index.js

```diff
@@ -19,8 +41,108 @@ class AuthButton extends Component {
   }
 }
 
+const MuiTableCell = withStyles({
+  root: {
+    borderBottom: "none",
+  }
+})(TableCell)
+
+
 
 export default class Profile extends Component {
+    state = {
+      groups: null,
+      newGroupName: "Running Buds",
+      newGroupTarget: 10,
+    }
+    constructor(props) {
+      super(props)
+      this.handleChangeName = this.handleChangeName.bind(this)
+      this.handleChangeTarget = this.handleChangeTarget.bind(this)
+      this.handleSubmit = this.handleSubmit.bind(this)
+    }
```

Comment:
@amn41 every time you use a class component instead of functional component a baby dolphin dies!!!